README.md

```diff
@@ -1,4 +1,748 @@
+Using the API, you can integrate Event Streams with any system that supports RESTful APIs.
+
+Documentation [IBM Cloud Eventstreams Service APIs](https://cloud.ibm.com/apidocs/event-streams).
+
+This is the Eventstreams Software Development Kit for `Go`
+It includes a library of functions used to access a Eventstreams cluster.
+
+Disclaimer: this SDK is being released initially as a **pre-release** version.
+Changes might occur which impact applications that use this SDK.
+
+## Table of Contents
+<!--
+  The TOC below is generated using the `markdown-toc` node package.
+
+      https://github.com/jonschlinkert/markdown-toc
+
+  You should regenerate the TOC after making changes to this file.
+
+      npx markdown-toc -i README.md
+  -->
+
+<!-- toc -->
+
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+    + [`go get` command](#go-get-command)
+    + [Go modules](#go-modules)
+    + [`dep` dependency manager](#dep-dependency-manager)
+- [Using the SDK](#using-the-sdk)
+- [REST API documentation](#event-streams-administration-rest-api)
+- [Questions](#questions)
+- [Issues](#issues)
+- [Open source @ IBM](#open-source--ibm)
+- [Contributing](#contributing)
+- [License](#license)
+
+<!-- tocstop -->
+
+## Overview
+
+The IBM Cloud Eventstreams Go SDK allows developers to programmatically interact with the following IBM Cloud services:
+
+Service Name | Package name 
+--- | --- 
+[Admin Rest](https://cloud.ibm.com/apidocs/event-streams) | AdminRest
+
+## Prerequisites
+
+* An [IBM Cloud](https://cloud.ibm.com/registration) account.
+* The [IBM Cloud CLI.](https://cloud.ibm.com/docs/cli?topic=cli-getting-started)
+* An IAM API key to allow the SDK to access your account. Create one [here](https://cloud.ibm.com/iam/apikeys).
+* A IBM Cloud Eventstreams Instance Create one [here](https://cloud.ibm.com/registration?target=/catalog/services/event-streams)
+* Go version 1.14 or above.
+
+## Installation
+The current version of this SDK: 1.0.0
+
+There are a few different ways to download and install the Eventstreams Go SDK project for use by your
+Go application:
+
+#### `go get` command  
+Use this command to download and install the SDK to allow your Go application to
+use it:
+
+```
+go get -u github.ibm.com/IBM/eventstreams-go-sdk 
+```
+
+#### Go modules  
+If your application is using Go modules, you can add a suitable import to your
+Go application, like this:
+
+```go
+import (
+	"github.ibm.com/IBM/eventstreams-go-sdk/pkg/adminrestv1"
+)
+```
+
+then run `go mod tidy` to download and install the new dependency and update your Go application's
+`go.mod` file.
+
+#### `dep` dependency manager  
+If your application is using the `dep` dependency management tool, you can add a dependency
+to your `Gopkg.toml` file.  Here is an example:
+
+```
+[[constraint]]
+  name = "github.ibm.com/IBM/eventstreams-go-sdk"
+  version = "0.0.1"
```

Comment:
Is the version `1.0.0` or `0.0.1`? The title say 1.0.0

Reply:
Coming soon !!! I am working on a PR that fixes the version number everywhere and will come from the swagger doc.
currently admin-rest is version 1.1.0 so the SDK will be 1.1.0